src/api/admin.ts

```diff
@@ -88,6 +89,14 @@ export type PlacementInput = {
 export const createUrlSafeName = (name: string) =>
   name.replace(/\W/g, "").toLowerCase();
 
+const getVenueOwners = async (venueId: string): Promise<string[]> => {
```

Comment:
Can we obtain this from the `currentVenue` and rely on snapshots, instead of getting it once and risking it being out-of-date?